client/src/resources/lang/en_us.json

```diff
@@ -461,136 +461,143 @@
+    "wiki.article.standard.priority.text":"Abilities are used in a specific order called priority order. If two players have abilities that go on the same priority, the player with the lower player number will go first. It is recommended that you read a roles encyclopedia page before this page. Some abilities go before night starts. Attacking someone doesn't kill them until the end of the night. Here are all the priorities and the abilities that happen in them.\n\n**Before Night (Not a priority)**\n\n- Jailor chooses who to jail\n- Journalist chooses who to interview\n- Spy finds if an apostle can convert\n- Death kills everyone\n- Create Visits\n  - All visits are decided by selecting players. When a roles ability activates on its priority, the ability is cast on who the visit is targeting, which is not necessarily who the player selected (in the case of a transporter, witch, necromancer, or retributionist).\n\n**1: Top Priority**\n\n- Amnesiac (Convert)\n- Doomsayer (Kill)\n- Executioner (Suicide)\n- Jester (Haunt)\n- Veteran (Decide to Alert)\n- Vigilante (Suicide)\n\n**2: Transporter**\n\n- Transporter (Swap Visits)\n\n**3: Control**\n\n- Witch (Possess)\n\n**4: Necromancy**\n\n- Retributionist (Possess/Change Visits)\n- Necromancer (Possess/Change Visits)\n\n**5: roleblock**\n\n- Hypnotist (roleblock)\n- Escort (roleblock)\n- Jailor (Invisible roleblock)\n\n**6: Deception**\n\n- Framer (Frame)\n- Blackmailer (Silence)\n- Janitor (Clean)\n- Hypnotist (Send Fake Messages)\n- Arsonist (Douse)\n\n**7: Bodyguard**\n\n- Bodyguard (Change visits)\n\n**8: Heal**\n\n- Bodyguard (Defend themselves)\n- Crusader (Defend target)\n- Guardian (Defend)\n- Veteran (Defend themselves)\n\n**9: Kill**\n\n- Arsonist (Ignite)\n- Bodyguard (Attack attacker)\n- Crusader (Attack Visitor)\n- Godfather (Attack or Make Backup Attack)\n- Jailor (Attack)\n- Mafioso (Attack)\n- Veteran (Rampage)\n- Vigilante (Attack)\n- Werewolf (Rampage and attacks)\n\n**10: Investigative**\n\n- Arsonist (Find who is doused)\n- Bodyguard (Find if protected)\n- Consigliere (Investigate)\n- Crusader (Find if protected)\n- Death (Collect souls)\n- Guardian (Find if protected)\n- Janitor (Find role and alibi)\n- Forger (Find role and alibi)\n- Lookout (Investigate)\n- Psychic (Investigate)\n- Seer (Investigate)\n- Detective (Investigate)\n- Spy (Find who the Mafia Visits)\n- Tracker (Investigate)\n- Werewolf (Track players scent)\n\n**11: Spy Bug**\n\n- Spy (Bug)\n\n**12: Steal Messages**\n\n- Witch (Get target messages)\n- Necromancer (Get target messages)\n- Retributionist (Get target messages)\n\n**13: Convert**\n\n- Cult (Convert)\n\n**14: SpyCultistCount**\n\n- Spy (Get Cult Count)",
```

Comment:
Trapper Reveler Zealot Apostle Consort Journalist (Post Journal) if needed psychic Politician Martyr

Reply:
Trapper:
heal - dismantle or set trap, build trap, use trap
kill- trap kill
Investigative - info messages, dismantle if should dismantle

reveler:
roleblock- roleblock

Zealot:
Kill - attack

Apostle
kill- attack
convert- convert

Consort
Top priority- ensure atleast one message
Roleblock- roleblock
deception- push messages

Journalist 
Investigative - post journal

Psychic
investigative- visions

Politician
all event based

Martyr
kill - kill